backend/game/engine/player_queue.py

```diff
@@ -0,0 +1,38 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = []
+        self.dead_players = []
+        self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
+        self.all_players.append(player)
+
+    def _remove_initial_player(self, player):
+        if player in self.all_players:
+            self.all_players.remove(player)
+
+    def set_player_order(self):
+        # TODO: set this based off of dice roll minigame
+        # Currently just the order in which they were added to the list
+        self.alive_players = self.all_players
+        self.player_cycle = itertools.cycle(self.all_players)
+
+    def get_current_player(self):
+        if not self.current_player:
```

Comment:
I don't quite understand this condition before returning the current player?

Reply:
This checks for a null, think on startup, so what happens is it’ll then cycle the next player which will be realistically the current player. If you called this on the next turn around, you would get the real current player as we don’t need to go through the cycle